invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -49,8 +49,6 @@ class Meta:
     # notes = fields.List(fields.Nested(InternalNoteSchema))
     revision = fields.Integer(dump_only=True)
 
-    status = fields.Str(dump_only=True)
-
     published = fields.Boolean(attribute='is_published', dump_only=True)
```

Comment:
Minor: See my comment earlier - about the name ``published`` vs ``is_published``. We need to align these two. Might be we just rename this one to ``is_published``.